parcel_test.go

```diff
@@ -31,57 +33,101 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	if err != nil {
```

Comment:
В тесте этот if можно полностью заменить на 
```suggestion
    require.NoError(t, err)
```

Reply:
Надо это переделать во всех тестах